itez/beneficiary/uuid.py

```diff
@@ -0,0 +1,11 @@
+import uuid
+
+
+def generate_id():
+    """
+     this function generate a 7 digit code for agent
+    """
+    uuid_one = uuid.uuid4()
+    print(uuid_one)
```

Comment:
we dont  need to print  it anywhere @cazterk 
lets get rid of this

Reply:
'print,' removed